src/Agent.Listener/CommandSettings.cs

```diff
@@ -132,13 +135,13 @@ public List<string> Validate()
             List<string> unknowns = new List<string>();
 
             // detect unknown commands
-            unknowns.AddRange(_parser.Commands.Where(x => !validCommands.Contains(x)));
+            unknowns.AddRange(_parser.Commands.Where(x => !validCommands.Contains(x, StringComparer.OrdinalIgnoreCase)));
```

Comment:
With the addition of the "localRun" subcommand we should more fully embrace ignore case for the command line arg names.

I also went through and changed the help to camel case all args. Seemed appropriate for readability.